File: Assignment_No-15/Question_04.c
/*
4)Accept N number from user and accept range Display aal elements from that range
N=6
start=60
end=90
elments=85 66 3 76 93 88
output=66 76 88
*/

#include<stdio.h>
#include<stdlib.h>
void Range(int Arr[],int iSize,int iStart,int iEnd)
{

	int iCnt=0;
	for(iCnt=0;iCnt<iSize;iCnt++)
	{

		if(Arr[iCnt]>=iStart && Arr[iCnt]<=iEnd)
		{

			printf("%d\n",Arr[iCnt]);
		}
	}
}
	

int main()
{



	int iLength=0;
	int iRet=0;
	int iCnt=0;
	int iValue1=0;
	int iValue2=0;
	int *ptr=NULL;

	printf("Enter  first   elments:\n");
	scanf("%d",&iLength);

	printf("Enter a second number:\n");
	scanf("%d",&iValue1);

	printf("Enter a third number:\n");
	scanf("%d",&iValue2);

    ptr=(int*)malloc(iLength*sizeof(int));

    if(ptr==NULL)
    {
    	printf("unable to allocate memory");
    	return -1;
    }
    printf("Enter elments",iLength);
    for(iCnt=0;iCnt<iLength;iCnt++)
    {
    		scanf("%d",&ptr[iCnt]);
    }

    Range(ptr,iLength,iValue1,iValue2);
    
    free(ptr);
    return 0;


}

File: Assignment_No-37/Question_04.java
/*4)write a program which accept N number from user and accept Range ,Display all elements from that range
input:N=6
start:60
End=90
elements:85 66 3 76 93 88
output=66 76 88 

*/

//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*; 
class Number
{
   public int Arr[];
	public Number(int iSize)
	{
		 Arr=new int[iSize];

    }
    public void Accept()
    {
    	int i=0;
    	Scanner sobj=new Scanner(System.in);
    	System.out.println("Enter the number of elments:");

    	for(i=0;i<Arr.length;i++)
    	{
    		Arr[i]=sobj.nextInt();
    	}
    }
    public void Display()
    {
        int i = 0;
        System.out.println("Elements of the array are : ");

        for(i = 0 ; i < Arr.length; i++)
        {
            System.out.println(Arr[i]);
        }

    }
    public void DisplayX(int iValue1,int iValue2)
    {
    	int i=0;

    	for(i=0;i<Arr.length;i++)
    	{
    		if(Arr[i]>=iValue1 && Arr[i]<=iValue2)
    		{
    			System.out.println(Arr[i]);
    		}
    	}
    }
  } 
 class ass194
{
	public static void main(String Arg[])
	{

        int iLength=0;
        int iStart=0;
        int iEnd=0;
        
		Scanner sobj=new Scanner(System.in);


		System.out.println("Enter the number:");
		iLength=sobj.nextInt();

		System.out.println("Enter the first number:");
		iStart=sobj.nextInt();

		System.out.println("Enter the last number:");
		iEnd=sobj.nextInt();

		Number nobj=new Number(iLength);
		nobj.Accept();
		nobj.Display();
		nobj.DisplayX(iStart,iEnd);
	}
		

}
